nextcord/guild.py

```diff
@@ -2804,6 +2804,32 @@ async def fetch_roles(self, *, cache: bool = False) -> List[Role]:
 
         return roles
 
+    async def fetch_role(self, role_id: int, /) -> Role:
+        """|coro|
+
+        Retrieves a :class:`Role` that a guild has.
```

Comment:
I understand you're following `fetch_roles` wording but to be picky I would prefer it to be similar to fetching single objects by ID, something like:

```suggestion
        Retrieve a :class:`Role` from this guild by its ID.
```
also matches the imperative form docstrings should have (which isn't true for all yet)

Reply:
Since this function under the `Guild` class, is specifying "guild by its ID" not a bit redundant?

Reply:
"A role the guild has" would also be redundant in that case, you can put what you like as there's no real convention yet, but imperative and descriptive is the idea